common/lib/dependabot/file_fetchers/base.rb

```diff
@@ -861,19 +861,35 @@ def decode_binary_string(str)
 
       sig { params(path: String).returns(T::Array[String]) }
       def find_submodules(path)
-        SharedHelpers.run_shell_command(
-          <<~CMD
-            git -C #{path} ls-files --stage
-          CMD
-        ).split("\n").filter_map do |line|
+        lfsEnabled = true
+        commandString = getCommandString(path,lfsEnabled)
+        /debugger
+        p commandString/
+        SharedHelpers.run_shell_command(commandString
+         ).split("\n").filter_map do |line|
```

Comment:
I think this could be simplified a bit without the helper methods and by using `Dir.chdir`

```suggestion
        lfs_enabled = # ...
        git_bin = lfs_enabled ? "git-lfs" : "git"
        Dir.chdir(path) do
          SharedHelpers.run_shell_command("#{git_bin} ls-files --stage")
        end.split("\n").filter_map do |line|
```